virtualizarr/parsers/kerchunk/parquet.py

```diff
@@ -90,8 +91,25 @@ def __call__(
         array_refs = {k: lrm[k] for k in lrm.keys()}
         full_reference = {"refs": array_refs}
         refs = KerchunkStoreRefs(full_reference)
+        # TODO: Deduplicate with KerchunkJsonParser
+        unique_paths = {
+            v[0]
+            for v in refs["refs"].values()
+            if isinstance(v, list) and isinstance(v[0], str)
+        }
+        registry = ObjectStoreRegistry()
+        for path in unique_paths:
+            parsed = urlparse(path)
+            if self.fs_root and not parsed.scheme:
+                registry.register(f"{self.fs_root}{path}", object_store)
+            elif not parsed.scheme:
+                raise ValueError(
+                    f"Detected relative path `{path}`. Parser must have `fs_root` set for "
+                    f"handling relative path. Parser has `fs_root = {self.fs_root}`."
+                )
+            else:
+                registry.register(path, object_store)
```

Comment:
these changes are not needed if we also adopt https://github.com/zarr-developers/VirtualiZarr/pull/662